Python/codegen.c

```diff
@@ -789,7 +830,17 @@ _PyCodegen_Expression(compiler *c, expr_ty e)
    and for annotations. */
 
 int
-_PyCodegen_Body(compiler *c, location loc, asdl_stmt_seq *stmts, bool is_interactive)
+_PyCodegen_Module(compiler *c, location loc, asdl_stmt_seq *stmts, bool is_interactive)
```

Comment:
Isn't it impossible to have an "interactive" module? I think the ast for an interactive toplevel node is ast.Interactive instead of ast.Module.

Reply:
Maybe it should be named "TopLevel" instead? I guess I think of interactive code as being basically like module-level code.

Reply:
TopLevel is at least very explicit and we know what we're handling. (I also needed to check that interactive nodes were indeed top-level nodes as I usually forget them, so I can understand why you can think of interactive code as module-level as well).